exercises/concept/temperature/.docs/instructions.md

```diff
@@ -0,0 +1,22 @@
+# Instructions
+
+You are part of a firefighter organization made of international volunteers. Over the last few calls your team has wasted valuable time translating from the new American team member charged with estimating the temperature inside the burning building. To save time, and people, you decide to create a program to automatically convert to and from Fahrenheit and Celsius for your new teammate.
```

Comment:
We have a "one sentence per line" style guide (see https://exercism.org/docs/building/markdown/markdown#h-one-sentence-per-line):

I've also suggested a slightly rephrased version, to not pin the blame on a specific group (Americans) :)

```suggestion
You are part of a firefighter organization made of international volunteers.
Due to the international nature of the team, valuable time has been wasted over the last few calls translating between Fahrenheit and Celcius while estimating the temperature inside the burning buildings.
To save time, and people, you decide to create a program to automatically convert between Fahrenheit and Celsius for your new teammate.
```

The main benefit of this, is that diffs become much easier to parse.